Source/Transport/TransportParams.H

```diff
@@ -4,9 +4,11 @@
 #include <AMReX_REAL.H>
 #include <AMReX_ParmParse.H>
 #include "TransportTypes.H"
+#include "PeleParamsGeneric.H"
 #include "EOS.H"
 
-namespace pele::physics::transport {
+namespace pele::physics {
+namespace transport {
```

Comment:
I think this is a clang tidy warning? Someone out there is telling us to `pele::physics::transport` instead of breaking it out? Or maybe I am missing something

Reply:
clang-tidy does require that nested namespaces be grouped when possible. However, it's not possible in this case so it's not an error. The transport namespace doesn't encompass everything in the file now. The InitParm template specializations have to be in the same namespace as the initial template definition, and that has to be the pele::physics namespace because the same template is used across different modules of PelePhysics.